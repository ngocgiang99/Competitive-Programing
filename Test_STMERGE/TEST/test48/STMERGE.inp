9
1 1
820257267 
1 7
33657983 99914120 545278357 424176750 874874089 504197307 558417549 
7 7
684325790 935060454 57345035 602177044 190532232 338314885 166013964 
366802591 1490958 576744846 140294923 990841535 915097860 155029606 
72021501 387414522 138488447 814435508 524357143 317047827 111941589 
832696923 839237837 652577553 563220634 472234880 504156118 47379256 
107316899 686823561 499739429 233381645 355464198 753059355 997760241 
622279509 920114967 867582747 274127277 438592789 487893099 445277357 
285189647 749353362 157250264 99984951 63773964 546220114 186532211 
9 5
503648024 745757071 52704262 755101163 243381739 
651940548 999506516 463747830 672838927 164796644 
699765935 999987953 231123209 519777798 177141690 
634981903 464688799 576819838 522919002 817571527 
477223636 287747298 569676726 416139724 375152878 
842092550 112842545 388188180 454266483 658377471 
406052976 574904532 554799399 480419107 326389081 
78077313 679617807 15988743 248155665 35377616 
835774576 716786558 37619965 337752502 334108111 
9 9
592998759 884362225 729321151 99267139 72030554 131473372 198913905 498722471 486730818 
208682196 567323789 707719507 310729493 567372893 344055077 218427372 483712777 67064382 
615246992 97473622 529648017 873781371 968744547 476542647 618461795 483442521 588245105 
418174836 936102259 103938398 710273234 619150641 825462008 817546573 28028376 205517524 
339591526 794396205 396946970 757113730 886291825 877655362 824860268 381179551 911403024 
429456913 609061860 632616401 752280694 974133855 365005270 795585788 63431705 380074033 
637236184 970093126 499595438 125065164 727565246 70766100 482713130 873426358 664756125 
51064075 446940866 348828850 793325002 846895119 895741675 231135175 444948700 73154629 
592094041 519238713 584191909 29581588 698062083 602087821 963855164 292601887 874928924 
10 1
159447883 
878650651 
303950383 
654090342 
337763007 
853589683 
739115976 
230856035 
40092893 
442749914 
2 3
33202765 337344709 499108896 
464547990 955179506 385697769 
4 2
655414514 929844730 
919770553 561840183 
83246257 147178991 
776634066 264610605 
5 10
136126906 580471277 672090947 427464341 70176375 816869389 743106592 296934512 16963404 761663329 
110056400 47212163 942895808 837181950 486566136 812195073 479360407 874187322 405698175 689336416 
787107542 82897066 155174279 215065429 26310594 152028705 824186744 892993748 718238688 969042358 
16490485 610886676 314489590 507552446 190696164 529507465 422336263 842094797 33283178 643328999 
22064730 959518136 604863723 365886500 562058099 856940949 809335565 996312438 791009047 96924344 
