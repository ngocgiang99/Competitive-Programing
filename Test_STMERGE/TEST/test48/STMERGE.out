820257267
33657983
119400757
243381739
215951283
159447883
418900534
347856862
22064730
